CourseSearchBuilderBlobTrigger/__init__.py

```diff
@@ -0,0 +1,65 @@
+import os

```

Comment:
General comments 

* I think the codebase is using spaces rather than tabs. For consistency I think we should use spaces - also spaces are preferred PEP8  method. 

* Also the codebase is, I think, using LFs as line terminators - so we don't have issues with diffs it might be best to change to use LF too.

In VSC I think you can set these in the bottom right of the screen. 

FWIW, I'm using flake8 for linting and yapf for formatting - both are pretty good.